add/Room + Inn module

buihieu028 opened the pull request:
Create Inn / Room class , repository + interface and creating new table in migrations

GuchiGangz committed:
add/Room + Inn module

Create Inn / Room class , repository + interface and creating new table in migrations

DinnieJ commented on api/database/migrations/2021_02_20_093313_create_room_table.php:
```diff
@@ -0,0 +1,43 @@
+            $table->float('price');
+            $table->float('acreage');
+            $table->text('description');
+            $table->tinyInteger('verified')->default(1);
```
verified ở đây default phải là false, vì trọ chưa được xác nhận

DinnieJ replied:
tinyInteger là trong databasae thôi, migration có cột boolean,
$talbe->boolean('verified')->default(0)

GuchiGangz committed:
Update 2021_02_20_093313_create_room_table.php

update cho bảng room table trường verified sang boolean với default false

GuchiGangz committed:
Update 2021_02_20_050809_create_inns_table.php

bổ sung trường location thêm nullable = true

GuchiGangz committed:
Add/OwnerSeeder và InnSeeder module

Bổ sung thêm 2 Seeder cho InnTable và OwnerTable, và những factory tương ứng cho 2 table trên
